### PR TITLE
Reader Conversations: remove pingbacks + trackbacks

### PR DESCRIPTION
**Pull request by samouri**
We should omit pingbacks + trackbacks from conversations tool.

addresses: https://github.com/Automattic/wp-calypso/issues/16920

### CODE REVIEW

**Comment by samouri** [comment]
Lint error I'd rather not fix: 
```
/home/ubuntu/wp-calypso/client/blocks/comments/post-comment.jsx
  288:30  error  React.PropTypes is deprecated since React 15.5.0, use the npm module prop-types instead  react/no-deprecated
```

This is the sort of thing where we should codemod it all away.

**Comment by samouri** [comment]
Okay, this 100% worked at hiding them. But the hidden trackback counted towards the three which we don't want. 

fixing.

**Comment by samouri** [comment]
fixed! theres still an issue where we don't load all the comments we should at once, but at least now the ui components are doing the right thing.

To test you should make sure a post doesn't display its trackbacks + pingbacks in the convo tool.  it'll display < 3 comments but if you click into the post and then click back you should see all 3 comments because more comments will have loaded.

We have a few options for how to proceed there but it should be a different PR

**Comment by bluefuton** [comment]
> theres still an issue where we don't load all the comments we should at once

Why does this happen? Is it worth us creating an issue for it while its fresh in your mind?

File: client/blocks/reader-post-card/conversation-post.jsx [hunk 1]
```diff
@@ -20,7 +20,16 @@ class ConversationPost extends React.Component {
...
 
 	render() {
-		const commentIdsToShow = map( take( this.props.comments, 3 ), 'ID' );
+		const commentIdsToShow = map(
+			take(
+				filter(
+					this.props.comments,
+					comment => comment.type !== 'trackback' && comment.type !== 'pingback'
+				),
+				3
```
Comment (bluefuton) [inline client/blocks/reader-post-card/conversation-post.jsx]: I know you didn't change it here ...but can we move this magic number to a var? `numberOfCommentsToShow`?

**Comment by samouri** [comment]
> Why does this happen? Is it worth us creating an issue for it while its fresh in your mind?

we load 3 in from es.  we aren't doing any querying/comment grabbing besides it.  we'll need to for various other pieces of this puzzle.  i'll make an issue

File: client/blocks/reader-post-card/conversation-post.jsx (hunk 1 above)
Comment (samouri) [inline client/blocks/reader-post-card/conversation-post.jsx]: 🆗

Comment (samouri) [inline client/blocks/reader-post-card/conversation-post.jsx]: actually, i'm looking at the code, id rather not waste time on it. 
we are going to need to rip all of the "how many comments to show" logic out anyway and I have no idea what that'll end up looking like.

should it be a prop? will this component itself be given all the ids? will everything be based off postKey? etc etc.

**Comment by fraying** [comment]
Works!

Before:

screen shot 2017-08-11 at 10 00 42 am


After:

screen shot 2017-08-11 at 10 00 36 am

**Pull request merged**